src/pages/messaging/agent-chat-api/index.mdx

```diff
@@ -1583,6 +1583,16 @@ https://api.livechatinc.com/v3.4/agent/action/delete_event_properties \
 
 ### Tag Thread
 
+Tags a given thread. The following restrictions apply:
+
+- Active and inactive chats can be tagged.
```

Comment:
let's drop this
you tag threads, not tags (just as the method name suggests 😉 )
and threads can be either active or not, there is no in-between, it's just like saying "you can tag all threads"

Reply:
chats -> threads 👍🏻
As for active and inactive, I specifically remember @bkubiak mentioning it as a valuable addition to this description

Reply:
For me, it is not as obvious as it may seem to be, I would stick to this info...I believe that with inactive threads it may not be so obvious for the user that he/she can also tag them - I would suggest changing to: "Active and inactive threads can be tagged" or "You can tag active and inactive threads"

Reply:
idk about @bkubiak's opinion but maybe you should add him to this review instead :trollface: 
for me this states "any thread" in a different way, and most of these methods are usable on any thread unless specified otherwise

Reply:
I don't know.... 😅  I still support my opinion that for some people possibility of adding tags to inactive threads may not be so obvious. 
@ale-kacperczyk WDYT? Maybe I just measure people's intelligence too low

Reply:
I wanted to add him for counter-balance but he's not available :) 

Jokes aside, I would continue to argue that this would be a helpful addition to novice developers, but this 👇 forces me to capitulate :)

> most of these methods are usable on any thread unless specified otherwise

@kamilatokarska, let's drop this line. Despite our good intentions, keeping the info could create some potential confusion as to why for some methods we specify that it works for active and inactive threads, and for other methods, we don't.